iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractHvdcConverterStation.java

```diff
@@ -57,6 +57,15 @@ public T setLossFactor(float lossFactor) {
         return (T) this;
     }
 
+    @Override
+    public HvdcConverterStation getOtherConverterStation() {
+        if (hvdcLine != null) {
+            return hvdcLine.getConverterStation1() == this ? hvdcLine.getConverterStation2() : hvdcLine.getConverterStation1();
+        } else {
+            throw new UnsupportedOperationException();
```

Comment:
I don't think we should throw an exception, maybe make the method return an optional that can be empty:
```java
public Optional<HvdcConverterStation> getOtherConverterStation() {
        if (hvdcLine != null) {
            return hvdcLine.getConverterStation1() == this ? Optional.of(hvdcLine.getConverterStation2()) : Optional.of(hvdcLine.getConverterStation1());
        } 
     return Optional.empty();
}
```